frontend/src/index.ts

```diff
@@ -466,6 +469,78 @@ document.addEventListener('DOMContentLoaded', function () {
   };
   nogoControl.addTo(map);
 
+  const provider =  new GoogleProvider({
+    params:{
+      key: 'AIzaSyDjcwjQH4vJuVF8c0EY73uDkYf1Xz4Ot1Y',
```

Comment:
obviously shouldn't put an API key in like this ... we can add an `.env` or something